src/julia.h

```diff
@@ -301,15 +299,14 @@ typedef struct _jl_method_instance_t {
     jl_value_t *rettype; // return type for fptr
     jl_svec_t *sparam_vals; // static parameter values, indexed by def.method->sparam_syms
     jl_array_t *backedges;
-    jl_value_t *inferred;  // inferred jl_code_info_t, or value of the function if jlcall_api == JL_API_CONST, or null
+    jl_value_t *inferred;  // inferred jl_code_info_t, or jl_nothing, or null
     jl_value_t *inferred_const; // inferred constant return value, or null
     size_t min_world;
     size_t max_world;
     uint8_t inInference; // flags to tell if inference is running on this function
-    uint8_t jlcall_api;
     uint8_t compile_traced; // if set will notify callback if this linfo is compiled
-    jl_fptr_t fptr; // jlcall entry point with api specified by jlcall_api
-    jl_fptr_t unspecialized_ducttape; // if template can't be compiled due to intrinsics, an un-inferred fptr may get stored here, jlcall_api = JL_API_GENERIC
```

Comment:
Yay :)

Reply:
Well, technically I just made generated functions more broken but just cared less about it. Although, as long as we make sure there's none included as part of basecompiler, I think it should not have any impact.

Reply:
So you're saying I have to abandon my plans for the new optimizer to have a dependency on Cxx.jl? Drat!

Reply:
I was just about to port codegen.cpp to LLVM.jl... /s

Reply:
Don't let that stop you, @maleadt 😃